src/store/flow/block.ts

```diff
@@ -110,9 +104,8 @@ export const mutations: MutationTree<IFlowsState> = {
     if (!block.ui_metadata) {
       Vue.set(block, 'ui_metadata', {})
     }
-    const base = block.ui_metadata!
 
-    let pointer = base
+    let pointer = block.ui_metadata!
```

Comment:
`base` wasn't used, so I simplified (and eslint was suggesting it as well, as I remember it)